Cython/Utility/ModuleSetupCode.c

```diff
@@ -1019,6 +1111,12 @@ static CYTHON_SMALL_CODE PyObject* ${pymodule_create_func_cname}(PyObject *spec,
     Py_DECREF(modname);
     if (unlikely(!module)) goto bad;
 
+#if CYTHON_COMPILING_IN_LIMITED_API
+    if (unlikely(__Pyx_copy_spec_to_module(spec, module, "loader", "__loader__") < 0)) goto bad;
```

Comment:
This doesn't look like it still needs duplication. Just assign `moddict = module` for the limited API.

Reply:
Good point!

Reply:
Fixed using the moddict assignment